action/protocol/staking/ethabi/v3/builder.go

```diff
@@ -0,0 +1,27 @@
+package v3
+
+import (
+	"encoding/hex"
+
+	"github.com/iotexproject/iotex-core/action/protocol"
+	stakingComm "github.com/iotexproject/iotex-core/action/protocol/staking/ethabi/common"
+)
+
+func BuildReadStateRequest(data []byte) (protocol.StateContext, error) {
```

Comment:
`BuildReadStateContext` is a better name?
also seems this func only used in test code in this PR?

Reply:
is used at next commit

Reply:
change the name? it's returning different new contexts